packages/bridge/contracts/l1/L1GovernanceRelay.sol

```diff
@@ -69,7 +81,7 @@ contract L1GovernanceRelay {
       targetData
     );
 
-    IInbox(inbox).createRetryableTicket{value: msg.value}(
+    IInbox(inbox).createRetryableTicket{value: l1CallValue}(
```

Comment:
I suggest either returning the inboxSequenceNumber that is returned here, or emit it in an event

Reply:
done by using L1CrossDomainEnabled